code/modules/spells/aoe_turf/fall.dm

```diff
@@ -72,7 +72,7 @@
 		invocation(user, targets)
 		take_charge(user, skipcharge)
 
-		targets = before_cast(targets)
+		targets = before_cast(targets, user)
```

Comment:
I don't know why it copies perform, I recently nixed another perform() version myself in my last pr